ext/opcache/jit/dynasm/minilua.c

```diff
@@ -7575,7 +7575,7 @@ size_t l=strlen(form);
 char spec=form[l-1];
 strcpy(form+l-1,"l");
 form[l+sizeof("l")-2]=spec;
-form[l+sizeof("l")-1]='\0';
+form[l+strlen("l")]='\0';
```

Comment:
Either keep the sizeof here, or I would imagine using ``strlen`` for the line above, but this looks a bit weird and might be done for good reasons